src/main/java/org/spongepowered/api/event/command/CommandProcessEvent.java

```diff
@@ -63,28 +94,4 @@
      */
     String getArguments();
 
-    /**
-     * Set the arguments as a string.
-     *
-     * <p>For example, if the message was {@code /example bob 3 -f}, then
-     * the arguments would be {@code bob 3 -f}.</p>
-     *
-     * @param arguments The arguments
-     */
-    void setArguments(String arguments);
-
-    /**
-     * The result of the command.
-     *
-     * @return The result of the command
-     */
-    CommandResult getResult();
-
-    /**
-     * Sets the result of the command.
-     *
-     * @param result The result of the command
-     */
-    void setResult(CommandResult result);
```

Comment:
What about `CommandResult.empty()`? Or maybe `CommandResult.cancelled()` (we'll have to create one)